pkg/apis/core/types.go

```diff
@@ -1966,6 +1966,33 @@ const (
 	PullIfNotPresent PullPolicy = "IfNotPresent"
 )
 
+// ContainerResizePolicy specifies user guidance on how container resource resize should be handled.
+// Only one of the following container resize policies may be specified.
+// If none of the following policies is specified, it defaults to NoRestart.
+type ContainerResizePolicy string
```

Comment:
Is this really "Policy" or "Requirement" ?

Reply:
Naming wise, I feel like this should be ResourceResizeRequirement or ResourceResizePolicy, and the struct below should be ContainerResizePolicy

Reply:
Thanks, I agree this looks cleaner. I'll update this to ResourceResizePolicy and ContainerResizePolicy, with values of Restart and NoRestart. RestartContainer got left that way because we had a third RestartPod option at one point which we dropped because we couldn't find a good documented use-case or reason to keep it in the IBM proposal (from where the resize policies came in the first place).